Tests/VaporFormsTests/VaporFormsTests.swift

```diff
@@ -700,5 +701,102 @@ class VaporFormsTests: XCTestCase {
       XCTAssertEqual(form.username, "user1")
     } catch { XCTFail() }
   }
+    
+    func testSampleLoginFormWithMultipart() {
+        // Test a simple login form which validates against a credential store.
+        struct LoginForm: Form {
+            let username: String
+            let password: String
+            static let fieldset = Fieldset([
+                "username": StringField(label: "Username"),
+                "password": StringField(label: "Password"),
+                ], requiring: ["username", "password"]) { fieldset in
```

Comment:
de-indent trailing `]` on line 713 and therefore subsequent lines in post-validation block